pages/linux/avahi-browse.md

```diff
@@ -0,0 +1,17 @@
+# avahi-browse
+
+> Displays services and hosts exposed on the local network via mDNS/DNS-SD.
+> Avahi is compatible with Bonjour (Zeroconf) found in Apple devices.
+> More information: <https://www.avahi.org/>.
+
+- List all services available on the local network along with their addresses and ports while ignoring local ones:
+
+`avahi-browse --all --resolve --ignore-local`
+
+- Limit your search to a particular domain:
+
+`avahi-browse --all --domain={{domain}}`
+
+- List all domains:
```

Comment:
I think this should be the second example, to let a user see all the domains first.

Reply:
Good idea. Looks like my review didn't go through, so I'll open a new one against this page with my suggestions too.

For the record, we generally try to avoid the 2nd person in tldr pages (e.g. you, your) @alufers :-)